fdbclient/SpecialKeySpace.actor.cpp

```diff
@@ -146,12 +146,14 @@ ACTOR Future<Standalone<RangeResultRef>> SpecialKeySpace::checkRYWValid(SpecialK
                                                                         KeySelector begin, KeySelector end,
                                                                         GetRangeLimits limits, bool reverse) {
 	ASSERT(ryw);
+	state Future<Void> clientTimeout = delay(5.0);
 	choose {
 		when(Standalone<RangeResultRef> result =
 		         wait(SpecialKeySpace::getRangeAggregationActor(sks, ryw, begin, end, limits, reverse))) {
 			return result;
 		}
 		when(wait(ryw->resetFuture())) { throw internal_error(); }
+		when(wait(clientTimeout)) { throw timed_out(); }
```

Comment:
I'm not exactly sure what the timeout here is measuring or what bug is being fixed, but this is a slightly different behavior from other client timeouts which are controlled via option and use a different error. I was curious if you had any thoughts about why one would be preferred over the other.

Reply:
I am sorry. I created this for a discussion with Andrew.
This is not a good choice here. I will delete them

Reply:
Oh, no worries. Sorry for jumping the gun.